roles/edpm_nova/tasks/configure.yml

```diff
@@ -31,7 +31,7 @@
     - {"path": "/var/lib/nova", "mode": "0755"}
     - {"path": "/var/lib/_nova_secontext", "mode": "0755"}
     - {"path": "/var/lib/nova/instances", "mode", "0755"}
-    - {"path": "/var/log/containers/nova", "mode": "0750"}
+    - {"path": "/var/log/containers/nova", "mode": "0755"}
```

Comment:
This should not be required, this folder will be owned by nova when the container starts https://github.com/openstack-k8s-operators/edpm-ansible/blob/c1344d96f00559a34794dedea489f6fe131b84fa/roles/edpm_nova/templates/config.json.j2#L38

We could premtivly change the owner to nova now that we create that user on the host but 755 is only needed for traversability of nested dirs. What requires that in this case?

Reply:
@jpodivin @SeanMooney 

In latest code, getting permission denied error due to config dir under path '/var/lib/openstack'.

config dir user is root and other user privilege  permission is 0. so we are not able to create any file inside '/var/lib/openstack/nova' dir in cloud-admin user. so need to use root permission for Render nova config files tasks.
or we need to change other user permission for config dir to 'x-r'.

[cloud-admin@edpm-compute-0 ~]$ ls -all /var/lib/openstack/
total 4
drwxr-xr-x.  6 root        root          71 Jan 10 08:30 .
drwxr-xr-x. 49 root        root        4096 Jan 10 08:30 ..
drwxr-xr-x.  3 cloud-admin cloud-admin   18 Jan 10 08:30 cacerts
drwxr-xr-x.  3 cloud-admin cloud-admin   18 Jan 10 08:30 certs
drwxr-x---.  5 root        root          48 Jan 10 08:30 config
drwxr-xr-x.  2 root        root          25 Jan 10 08:16 reboot_required
[cloud-admin@edpm-compute-0 ~]$ ls -all /var/lib/openstack/config/
ls: cannot open directory '/var/lib/openstack/config/': Permission denied